doc/changes/latest.inc

```diff
@@ -78,6 +80,10 @@ Enhancements
 
 - Support new EEGLAB file format (:gh:`8874` by `Clemens Brunner`_)
 
+API changes
+~~~~~~~~~~~
+- ``mne.read_selection`` has been deprecated in favor of `mne.read_vectorview_selection`. ``mne.read_selection`` will be removed in MNE-Python 0.24 (:gh:`8870` by `Richard Höchenberger`_)
+
```

Comment:
Actually no it's still in the wrong spot

Reply:
OUCH! Arrgh, sorry about that, will fix.

Reply:
Should be good now.